sdk/formrecognizer/ai-form-recognizer/src/transforms.ts

```diff
@@ -300,6 +316,7 @@ export function toFormFromPageResult(original: PageResultModel, pages: FormPage[
 export function toRecognizedForm(original: DocumentResultModel, pages: FormPage[]): RecognizedForm {
   return {
     formType: original.docType,
+    modelId: original.modelId,
```

Comment:
should formTypeConfidence be added here too or did I miss it from somewhere else?

Reply:
:pray: thank you for pointing this out. Something happened here with the commits and I lost this and the mapper for `selectionMark` values as well as tests for formTypeConfidence. I'll patch those up.